File: Data Structures & Algorithms/is-anagram/submission-14.swift
class Solution {
    func isAnagram(_ s: String, _ t: String) -> Bool {
        var hashMap: [Character: Int] = [:]
        var hashMapTwo = [Character:Int]()

        for i in s {
            hashMap[i, default: 0] += 1
        }

        for i in t {
            hashMapTwo[i, default: 0] += 1
        }

        return hashMap == hashMapTwo
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-14.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        hashMap = {}

        for idx, val in enumerate(nums):
            if target - val in hashMap:
                if idx <= hashMap.get(target - val, 0):
                    return [idx, hashMap.get(target - val, 0)]
                else:
                    return [hashMap.get(target - val, 0), idx]
            hashMap[val] = idx